### DISCUSSION TITLE
An Idea

### DISCUSSION THREAD

**Discussion by ghost** [d2phap/ImageGlass#1367] [General]
An Idea for ImageGlass

Hi, @d2phap 
You know **Windows Photo Viewer** from Windows 7 which is still Available in Windows 10 and it's Very Fast Compare the any other Program, but Microsoft Remove it's Registry Keys, So The Program Not Accessible without adding Registry Keys..

If You Try to Reverse Engineering of Windows Photo Viewer, The You get Many New Ideas for Improving & Speedup ImageGlass..

Thank You!

**Comment by d2phap** [comment]
ImageGlass is far different from the Photo viewer about the features, framework, programming language,... ImageGlass also supports color management which makes it a bit slower when it's enabled, and other auto-corrections like rotation, reading image info, EXIF profile,... which the Photo viewer does not have at all.

**Comment by ghost** [comment]
I already know the difference between **Windows Photo Viewer & ImageGlass**. The main reason, why I use ImageGlass

But Speed, When Double Click on An Image
- ImageGlass Takes 2-3 Seconds Everytime
- Windows Photo Viewer Starts Before Start Counting 1 Second

If You Try To Find Witch Framework, Technique Use In **Windows Photo Viewer (Which is Default in Windows 7)** You Will Get Some Ideas Definitely..
